frontend/demo/README.md

```diff
@@ -49,4 +49,13 @@ In order to communicate with our API endpoints, you need a valid [JWT](https://j
 
 Aside from Curl, [PostMan](https://www.postman.com/downloads/) and other API testing tools could also be used to access the endpoints.
 
+### Development
+
+To start the app in development mode, run these commands:
+
+```
+yarn
+yarn ibazel run //frontend/demo:bundle_server
```

Comment:
Is the second `yarn` necessary? 🤔

Reply:
yes, unless you install `ibazel` globally. Which i don't want to advocate because most users won't use it outside of the airy context.